lib/awestruct/handlers/template/asciidoc.rb

```diff
@@ -19,7 +19,7 @@ def prepare
       end
 
       def evaluate(scope, locals, &block)
-        @output ||= ::Asciidoctor.render(data, options, &block)
+        @output ||= (scope.document = ::Asciidoctor.load(data, options, &block)).convert
```

Comment:
The &block parameter is ignored by Asciidoctor, so there's no need to pass
it on.